### ISSUE TITLE
Empty alert message in alert table for most bundled alerts

### ISSUE THREAD

**Issue by bashofmann** [rancher/dashboard#4883]
**Setup**
- Rancher version: 2.6.3
- Browser type & version: Chrome

**Describe the bug**
The alert table on the cluster dashboard and on the monitoring dashboard shows an empty for most of the alerts that are bundled with rancher-monitoring.

**To Reproduce**
Install rancher-monitoring

**Result**
The message column in the alert table is empty for most of the alerts, including the Watchdog alert which is always firing.

**Expected Result**
The message column is populated with the alert message.

**Screenshots**
![Bildschirmfoto 2022-01-06 um 19 09 38](https://user-images.githubusercontent.com/243056/148444809-06b13a5e-f8ef-43b0-a390-85400d5e54a0.png)

**Additional context**
The UI looks for the message in a `message` annotation on the firing alert.

From the alerts bundled with rancher-monitoring only 13 contain a `message` annotation, `111` have the alert message in a `description` annotation.

```
❯ kubectl get prometheusrules -A -o yaml | grep message: | wc -l
13

❯ kubectl get prometheusrules -A -o yaml | grep description: | wc -l
111
```

**Comment by nwmac** [comment]
This is probably broken due to the model changes - @richard-cox can hep you here.

**Comment by bashofmann** [comment]
I don't think it's due to a model change. Annotations in PrometheusRules are not really standardised. Some use the `message` annotation for a human readable alert message, a lot use `description`. Prometheus/Alertmanage just return all annotations defined in the PrometheusRules for firing alerts.

**Comment by nwmac** [comment]
@aalves08 Sorry - there's a PR up for this already - can you review and merge as appropriate - https://github.com/rancher/dashboard/pull/4884

**Comment by aalves08** [comment]
Connected to PR https://github.com/rancher/dashboard/pull/4884 which fixes this issue.

**Comment by ronhorton** [comment]
**PASS** Verified in `2.6-head Commit ID da382a8`

1. Install rancher monitoring
2. check Active Alerts on `Monitoring` page

Fix in 2.6-head
<img width="703" alt="Screen Shot 2022-02-14 at 2 32 18 PM" src="https://user-images.githubusercontent.com/89212856/153933583-d48047a1-dc53-4c1b-b6cd-c68e8fdd38aa.png">

Issue as filed, reproduced in 2.6.3
<img width="997" alt="Screen Shot 2022-02-14 at 2 31 21 PM" src="https://user-images.githubusercontent.com/89212856/153934218-d3ea92bb-d69f-4b58-a472-f681ece6ece4.png">
